Libraries/oneDAL/daal4py_Distributed_LinearRegression/License.txt

```diff
@@ -0,0 +1,20 @@
+  
```

Comment:
Is there a reason that you are not using the standard onePI License file? You should use this one: https://github.com/oneapi-src/oneAPI-samples/blob/master/License.txt

Reply:
fixed.